images/custom-scorecard-tests/cmd/test/main.go

```diff
@@ -78,10 +77,52 @@ func printValidTests() scapiv1alpha3.TestStatus {
 	result.Suggestions = make([]string, 0)
```

Comment:
IHMO all code in the internal/ dir should be related an internal implementation of the CLI features. We should not add examples inside that since they are not really internal.